CSVSource/build.gradle

```diff
@@ -42,10 +42,9 @@ dependencies {
     
     compile "org.slf4j:slf4j-api:1.7.25"
     compile "org.apache.logging.log4j:log4j-slf4j-impl:2.11.0"
-    
-      
-    // Used for checking VANTIQ date format behavior with VANTIQ SDK
-    compile 'io.vantiq:vantiq-sdk:1.0.27'
+
+    // Used in tests
+    testCompile 'io.vantiq:vantiq-sdk:1.0.28'
```

Comment:
I would suggest that for all of these where we want thing to be the same one, that we add an ext variable (`vantiqSdkVersion = 1.0.28`) to the top level `build.gradle` file, and then reference that in all the subproject gradle files.  Then, when this changes to ...29, etc., we only have to change one place.  Like we do for `${jacksonVersion}` in extjsdk.

Note that all the includes that use single ' will have to change to double " to cause the appropriate GString evaluation.